scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,262 @@
+        [ skos:example "stopIfTTCLowerThan" ;
+            sh:datatype xsd:string ;
+            sh:description "Scenario/domain specific commands or actions"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of customCommands failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "customCommands"@en ;
+            sh:order 6 ;
+            sh:path scenario:customCommands ] ;
+    sh:targetClass scenario:Participants .
+
+scenario:StructuralShape a sh:NodeShape ;
+    sh:property [ skos:example "http://mywebsite.de/Catalogs" ;
+            sh:description "Link to Catalogs"@en ;
+            sh:message "Validation of catalogs failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "catalogs"@en ;
+            sh:node scenario:LinkShape ;
+            sh:order 0 ;
+            sh:path scenario:catalogs ],
+        [ skos:example "http://mywebsite.de/myModel.fbx" ;
+            sh:description "Link to 3D models"@en ;
+            sh:message "Validation of 3dModels failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "3dModels"@en ;
+            sh:node scenario:LinkShape ;
+            sh:order 1 ;
+            sh:path scenario:3dModels ],
+        [ skos:example "http://mywebsite.de/map.xodr" ;
+            sh:description "Link to traffic space"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of trafficSpace failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "trafficSpace"@en ;
+            sh:node scenario:LinkShape ;
+            sh:order 2 ;
+            sh:path scenario:trafficSpace ] ;
+    sh:targetClass scenario:Structural .
+
+scenario:TrafficShape a sh:NodeShape ;
+    sh:property [ skos:example "kangaroo crossing" ;
+            sh:datatype xsd:string ;
+            sh:description "Country specific traffic signs"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of countrySpecificSign failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "countrySpecificSign"@en ;
+            sh:order 3 ;
+            sh:path scenario:countrySpecificSign ],
+        [ skos:example "tuk tuk" ;
+            sh:datatype xsd:string ;
+            sh:description "Country specific traffic participants"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of countrySpecificTP failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "countrySpecificTP"@en ;
+            sh:order 2 ;
+            sh:path scenario:countrySpecificTP ],
+        [ skos:example "germany, not applicable" ;
+            sh:datatype xsd:string ;
+            sh:description "Country where this scenario is from"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of country failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "country"@en ;
+            sh:order 0 ;
+            sh:path scenario:country ] ;
+    sh:targetClass scenario:Traffic .
+
+scenario:LinkShape a sh:NodeShape ;
```

Comment:
as far as I can see the LinkShape is already there in `general`? Or is this something different?

Reply:
This is right, fixed in f19ba90cd1a8a55ce44f67212592d88f69dd00d4